angularjs-portal-frame/src/main/webapp/portal/main/partials/header.html

```diff
@@ -24,13 +24,16 @@
              </div>
              <ul class="nav navbar-nav hidden-xs bucky-nav-large">
                  <li>
-                    <notification-bell></notification-bell>
+                    &nbsp;
```

Comment:
Is this for an extra space someplace?  Would adding padding somewhere (search directive) be preferable to using `&nbsp` for layout?  Or is this for something else besides layout?